Utilities/docker

```diff
@@ -0,0 +1,163 @@
+        "swiftpm-docker-1604"
+    ] + command_args)
+
+def build(_):
+    """Builds a docker image with the latest snapshot."""
+
+    # Get latest snapshot info.
+    latest_build_url = "https://swift.org/builds/development/ubuntu1604/latest-build.yml"
+    latest_build_data = yaml.load(urllib2.urlopen(latest_build_url).read())
+    snapshot_filename = latest_build_data["download"]
+    # FIXME: We shouldn"t need to do this, it should be available in the API.
+    snapshot_name = snapshot_filename.replace("-ubuntu16.04.tar.gz", "")
+    base_url = latest_build_url.rsplit("/", 1)[0]
+    latest_snapshot_url = base_url + "/" + snapshot_name + "/" + snapshot_filename
+    script_dir = os.path.dirname(os.path.realpath(__file__))
+
+    # Download latest snapshot (if necessary).
+    if snapshot_filename in os.listdir(script_dir):
+        print "Not downloading: " + latest_snapshot_url
+    else:
+        result = call([
+            "curl",
+            "-o",
+            script_dir + "/" + snapshot_filename,
+            latest_snapshot_url
+        ])
+
+        if result != 0:
+            print "Unable to install package: " + latest_snapshot_url
+            exit(1)
+
+    # Create docker image.
+    call([
+        "docker",
+        "build",
+        "-t", "swiftpm-docker-1604",
+        "--build-arg", "SNAPSHOT=%s" % snapshot_filename,
+        script_dir
+    ])
+
+def run(args):
+    """Runs a command in the container."""
+    docker_run([args.command] + args.arguments)
+
+def bootstrap(args):
+    """Runs the bootstrap script in the container."""
+    docker_run(["Utilities/bootstrap"] + args.arguments)
+
+def swift_build(args):
+    """Runs the swift-build executable in the container."""
+    docker_run([".build/x86_64-unknown-linux/debug/swift-build"] + args.arguments)
+
+def swift_test(args):
+    """Runs the swift-test executable in the container."""
+    docker_run([".build/x86_64-unknown-linux/debug/swift-test"] + args.arguments)
+
+def swift_run(args):
+    """Runs the swift-run executable in the container."""
+    docker_run([".build/x86_64-unknown-linux/debug/swift-run"] + args.arguments)
+
+def main():
+    """Main script entry-point."""
+
+    parser = argparse.ArgumentParser(
+        usage="%(prog)s [build|run|bootstrap|swift-build|swift-test|swift-run]",
+        description="This script simplifies all the docker operations to build "
+                    "and run a container for SwiftPM development and testing "
+                    "on Linux.")
+    subparsers = parser.add_subparsers()
+
+    # build
+    parser_build = subparsers.add_parser(
+        "build",
+        help="builds a docker image from the latest snapshot.")
+    parser_build.set_defaults(func=build)
+
+    # run
+    parser_run = subparsers.add_parser(
+        "run",
+        help="runs a command in a container.")
+    parser_run.add_argument("command", help="the command to run")
+    parser_run.add_argument("arguments", nargs=argparse.REMAINDER)
```

Comment:
This should be `--arguments`.

Reply:
I tried that already but it doesn't resolve the error.

Reply:
Works for me, did you add it to all argument decls?

Reply:
I pushed a new version with all argument decls as `--arguments` but still doesn't work. Am I missing something obvious?